src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/EndreBehandlendeEnhet/EndreBehandlendeEnhet.tsx

```diff
@@ -78,47 +123,10 @@ const EndreBehandlendeEnhet: React.FC<IProps> = ({ ytelse, behandling, onListEle
                             size="small"
                         >
                             Bekreft
-                        </FTButton>,
-                    ],
-                }}
-            >
-                <>
-                    <FamilieSelect
-                        {...skjema.felter.enhet.hentNavInputProps(skjema.visFeilmeldinger)}
-                        erLesevisning={false}
-                        name="enhet"
-                        label={'Velg ny enhet'}
-                        value={skjema.felter.enhet.verdi}
-                        onChange={e => skjema.felter.enhet.validerOgSettFelt(e.target.value)}
-                    >
-                        <option value={''} disabled={true}>
-                            Velg ny enhet
-                        </option>
-                        {behandendeEnheter.map(enhet => (
-                            <option key={enhet.enhetId} value={enhet.enhetId}>
-                                {enhet.enhetNavn}
-                            </option>
-                        ))}
-                    </FamilieSelect>
-                    <Spacer8 />
-                    <FamilieTilbakeTextArea
-                        {...skjema.felter.begrunnelse.hentNavInputProps(skjema.visFeilmeldinger)}
-                        label={'Begrunnelse'}
-                        erLesevisning={false}
-                        value={skjema.felter.begrunnelse.verdi}
-                        onChange={e => skjema.felter.begrunnelse.validerOgSettFelt(e.target.value)}
-                        maxLength={400}
-                    />
-                    {feilmelding && (
-                        <>
-                            <Spacer8 />
-                            <div className="skjemaelement__feilmelding">
-                                <ErrorMessage size="small">{feilmelding}</ErrorMessage>
-                            </div>
-                        </>
-                    )}
-                </>
-            </UIModalWrapper>
+                        </FTButton>
```

Comment:
Her må vi bytte om rekkefølgen på knappene for at de skal dukke opp likt som før 😄

Reply:
Rekkefølge er byttet.